src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,34 @@
 'use strict';
 
 class Animal {
-  // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+
+    Animal.alive.push(this);
+  }
 }
 
+Animal.alive = [];
+
 class Herbivore extends Animal {
-  // write your code here
+  constructor(name) {
+    super(name);
+    this.hidden = false;
+  }
+  hide() {
+    this.hidden = true;
+  }
```

Comment:
```suggestion
  hide() {
    this.hidden = !hidden;
  }
```

**changes the hidden property of the beast to the opposite value**

I guees changing to opposite means that you can hide for a period of time, which sets hidden to true, then you can call this method again and stop hiding, which sets hidden to opposite, hidden = !hidden

Reply:
agree